content/tutorials/webrtc/basics/en/index.html

```diff
@@ -343,43 +344,43 @@ <h2 id="toc-signaling">Signaling: session control, network and media information
 
 <p>The acquisition and exchange of network and media information can be done simultaneously, but both processes must have completed before audio and video streaming between peers can begin.</p>
 
-<p>The offer/answer architecture described above is called <a href="http://tools.ietf.org/html/draft-ietf-rtcweb-jsep-00" title="IETF JSEP draft proposal">JSEP</a>, JavaScript Session Establishment Protocol. (There's an excellent animation explaining the process of signaling and streaming in <a href="http://www.ericsson.com/research-blog/context-aware-communication/beyond-html5-peer-peer-conversational-video/" title="Ericsson conversational video demo">Ericsson's demo video</a> for its first WebRTC implementation.) </p>
+<p>The offer and answer architecture previously described is called <a href="https://tools.ietf.org/html/draft-ietf-rtcweb-jsep-00" title="IETF JSEP draft proposal">JSEP</a>, or JavaScript Session Establishment Protocol. (There's an excellent animation explaining the process of signaling and streaming in <a href="https://www.ericsson.com/research-blog/context-aware-communication/beyond-html5-peer-peer-conversational-video/" title="Ericsson conversational video demo">Ericsson's demo video</a> for its first WebRTC implementation.) </p>
```

Comment:
I think the link should now be https://rtcweb-wg.github.io/jsep/

Reply:
Will change